.support/pypi_vs_conda_names.json

```diff
@@ -14,5 +14,6 @@
   "pyiron-potentialfit": "pyiron_potentialfit",
   "pyiron-workflow": "pyiron_workflow",
   "tables": "pytables",
-  "landaupkg": "landau"
+  "landaupkg": "landau",
```

Comment:
Isn't it simply `landau` for both? [Conda](https://anaconda.org/conda-forge/landau) is fine, and on PyPI [landau](https://pypi.org/project/landau/) looks more up-to-date than [landaupkg](https://pypi.org/project/landaupkg/)